requirements.txt

```diff
@@ -0,0 +1,5 @@
+-e .
+
+black>=20.8b1
+mypy==0.780
+flake8>=3.8.0
```

Comment:
do you want to install all the same flake8 plugins as are in the main repo requirements.txt?

Reply:
Sure, do you have to also enable them somewhere, or is having them installed sufficient?